### ISSUE TITLE
Suggestions for finding veins

### ISSUE THREAD

**Issue by iRedemption** [tom5454/Create-Ore-Excavation#119]
Love this mod but I think its biggest weakness is finding the veins since it can be quite tedious. Here are some suggestions that could be considered:

1. Showing what direction the vein is. This could be done by showing direction in text (North, West, South, East), similar to the distance. Even better, the vein finder could be a compass so it just shows the direction on the texture and the rest of the functionality could remain the same.

2. Add a second tier, more expensive vein finder that lets you choose which type of vein you want to find, similar to explorer's compass but for veins. This could be incredibly useful especially for modpacks that have a lot of ores or have them whitelisted in only certain biomes.

3. Add Xaero's Map support. I don't know if Xaero's map supports similar features of journey map but if you have the advanced finder it could just add a waypoint on the vein location

Some other ideas that came up:

1. The ability to require more than one type of liquid to drill.

2. Make ore vein infinite only on certain drill types. Lower quality drills slowly 'destroy' the ore vein while higher quality ones depleet it more and more slowly until the player uses a netherite drill (or a more expensive modded one) which makes it infinite. 

3. Be able to refill the vein to continue drilling. This could be done by having to manually right click the drill with a craftable upgrade to only after the vein has been depleted that lets the drill go 'deeper' to find more ores. The deeper you go the higher the multiplier of the yield/vein base amount?

**Comment by ropi336** [comment]
Yeah this desperately needs some more usable type of finder, the current ore finder is very bad especially if you want to find something like netherite
